### PR TITLE
Add clean-docker script

### PR DESCRIPTION
**Pull request by robert-damoc**
I was wondering if we want this script since it removes all the containers, and the dangling images (which includes all the images and containers in the system, not just ours).

One of the things we might want to add is a text prompt, to ask the user to confirm if he wants to proceed with this, but maybe we want to also do some extra work, to minimize the effect on other images/containers, and touch only the ones we are using.

### CODE REVIEW

**Comment by CodyPredy** [comment]
> I was wondering if we want this script since it removes all the containers, and the dangling images (which includes all the images and containers in the system, not just ours).
> 
> One of the things we might want to add is a text prompt, to ask the user to confirm if he wants to proceed with this, but maybe we want to also do some extra work, to minimize the effect on other images/containers, and touch only the ones we are using.

I see it more as an added bonus since we're probably going to run them simply by docker-compose up initially. 

Since it's a separate script file you can go wild :D. The only issue is that we're getting deeper into devops territory.

**Comment by robert-damoc** [comment]
I mean, yeah we can go wild, but if you give too much power into the wrong hands, you can have a bad result :D

File: bin/clean-docker.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+running_containers=$(docker ps -a | wc -l)
+
+# Check if any containers are running
+#     1 - no running containers
+#   > 1 -    running containers
```
Comment (intojs) [inline bin/clean-docker.sh]: OCD :) i like it! 💯

File: bin/clean-docker.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+if [ $running_containers -ne 1 ]; then
+  # Stop running containers
+  docker stop $(docker ps -a | tail -n+2 | awk '{ print $1 }')
+  # Remove containers
+  docker rm $(docker ps -a | tail -n+2 | awk '{ print $1 }')
+fi
```
Comment (intojs) [inline bin/clean-docker.sh]: Everything here is  gibberish for me :) So it's great! 💯

**Pull request merged**